### DISCUSSION TITLE
Pregabalin + Tramadol

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#91] [Drug Interactions]
Research indicates that this combination is well-tolerated. However, there may be a slightly increased risk of uncomfortable side effects, such as dizziness and vomiting.

Lee, S., Kim, Y., Lee, J. J. S., Im, G., Cho, J.-Y., Chung, J.-Y., & Yoon, S. (2018). A pharmacokinetic drug-drug interaction study between pregabalin and tramadol in healthy volunteers. European Journal of Clinical Pharmacology, 74(12), 1605–1613. https://doi.org/10.1007/s00228-018-2543-0

**Comment by utaninja** [comment]
I'd be concerned about seizure concerns being raised by the combination of these drugs. Still unsure about pregabalin's seizure concerns, unfortunately. Am actively looking into it.

**Reply by zelixir25** [reply]
yea that would be good to get more concrete info. i think at the very least a caution classification would be appropriate. something like "While this combination may be well-tolerated at common therapeutic dosages, there may be an increased risk of seizures and other adverse effects at higher doses. Use caution when combining these drugs."? dunno will have to keep looking.

also here's a paper on pregabalin toxicity based on dose. i know you've been doing a bunch of research on pregabalin already so you may have seen this before.

Rietjens, S. J., Sikma, M. A., Hunault, C. C., de Lange, D. W., & Hondebrink, L. (2022). Pregabalin poisoning: Evaluation of dose‐toxicity relationship. British Journal of Clinical Pharmacology, 88(3), 1288–1297. https://doi.org/10.1111/bcp.15073

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/293
